sharktank/sharktank/layers/causal_llm.py

```diff
@@ -95,10 +95,9 @@ def input_mask(
 
     def decode_attention_mask(self, boolean_input_mask: torch.Tensor):
         dtype = self.attention_dtype
-        numeric_mask = torch.zeros_like(boolean_input_mask, dtype=dtype)
-        numeric_mask.masked_fill_(
-            boolean_input_mask, self._maximally_negative_value(dtype)
-        )
+        numeric_mask = torch.where(
```

Comment:
I'm not opposed to a war, but it seems like we should fix something here at it's core... Or at least file an issue

Reply:
Going to write a fix for this right now. Just need to land the workaround to avoid people being blocked.